<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>A9Class</class>
 <widget class="QWidget" name="A9Class">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>A9</string>
  </property>
  <widget class="QStackedWidget" name="stackedWidget">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>30</y>
     <width>761</width>
     <height>561</height>
    </rect>
   </property>
   <widget class="QWidget" name="page">
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>200</x>
       <y>70</y>
       <width>391</width>
       <height>131</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Times New Roman</family>
       <pointsize>24</pointsize>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>CHOOSE REPOSITORY</string>
     </property>
    </widget>
    <widget class="QWidget" name="horizontalLayoutWidget">
     <property name="geometry">
      <rect>
       <x>170</x>
       <y>190</y>
       <width>421</width>
       <height>191</height>
      </rect>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QPushButton" name="CSV">
        <property name="font">
         <font>
          <bold>true</bold>
         </font>
        </property>
        <property name="text">
         <string>CSV</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="HTML">
        <property name="font">
         <font>
          <bold>true</bold>
         </font>
        </property>
        <property name="text">
         <string>HTML</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="textFile">
        <property name="text">
         <string>TEXT FILE</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="MEMORY">
        <property name="font">
         <font>
          <bold>true</bold>
         </font>
        </property>
        <property name="text">
         <string>MEMORY</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </widget>
   <widget class="QWidget" name="page_2"/>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="A9.qrc"/>
 </resources>
 <connections/>
</ui>
